package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "mijn-amsterdam-frontend",
-  "version": "4.100.0",
```

Comment:
geen versies meer in packagejson?

Reply:
Nee idd. Dat zag ik bij SIA ook. Die versie updates zijn lastig omdat er een PR gemaakt moet worden voor alleen package JSON. Vervolgens komt de release tag niet mee oid. Anyway, lastig en de git tag zou genoeg moeten zeggen.